### DISCUSSION TITLE
[Question]: Sydney does not remember previous messages in the chat?

### DISCUSSION THREAD

**Discussion by Cascaisdude** [danny-avila/LibreChat#1128] [Q&A] [answered]
### Contact Details

markpeartree@protonmail.com

### What happened?

While the normal Bing AI chat is working well with no bugs at all, the jailbroken Sydney does not seem to remember any of the messages in the chat. In fact, it seems that every time you send a message through the chat with Sydney, she acts like it's a brand new chat every single time, I say that because she would introduce herself in every reply and for a while every single reply she made would change the title of the chat, while normally the title of the chat would only be changed by the first messages in a new conversation. As far as I know, the issue doesn't seem to be the cookie string token, I messed with the .env file to no avail.

This problem is similar to another problem some other user had, but this issue is only with Sydney, normal Bing chat doesn't have any problems. I didn't find anything else regarding this type of issue, but if something like this was already posted and solved I apologize, I didn't look further enough then.

### Steps to Reproduce

1. Tell your name to Sydney
2. Then ask her if she remembers your name

She'll always answer a variation of "Of course I do! It's <completely random name or 'user'>, you told me before!" Of course, while also reintroducing herself again and again...

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
Hi @Cascaisdude 

Thanks for your report. I've had to move it to a discussion because there may not be one underlying issue that we can really tackle.

Also be sure to update to the latest commits of the project to assure things are working as intended. Any of the installation guides here can walk you through this: https://docs.librechat.ai/install/docker_compose_install.html

Technically, every new message using Sydney mode creates a new conversation. It is always using new conversations to string as much of a jailbroken response together as it can. The output of the LLM cannot be guaranteed as results will vary. If something is teetering its "memory," one or all of these conversations are triggering a blocker, whether it's a censor, IP filter, rate limit, etc. 

As you can see, I cannot reproduce what you reported even now:
![image](https://github.com/danny-avila/LibreChat/assets/110412045/8b4fccb4-7c1c-48e2-835e-12bad10ec612)

**Reply by Cascaisdude** [reply]
Thank you for such a quick response @danny-avila and yes I suspected it might be something on my side, maybe a network issue. I used that bingai pro extension briefly some time ago and I faced the same issue, so I was inclined to think that this wasn't a LibreChat issue, but I don't know how that extension was made, maybe they used LibreChat with it. Anyway thanks again for your reply, do you have any steps I could follow to try and solve the issue? I'm going to make a brand new installation of LibreChat, refresh the cookie token, change browsers and if everything else fails, I'm changing my bing account, since that account's tokens might be bugged or something, and I faced the same issue with the bingai extension within my account.
